controllers/itemController.js

```diff
@@ -0,0 +1,111 @@
+const Item = require("../models/itemsModel/item_model");
+const { StatusCodes } = require("http-status-codes");
+
+const ItemController = {
+  // Retrieve all items
+
+  getItems: async (req, res) => {
+    await Item.sync(); // This creates the table if it doesn't exist
```

Comment:
We dont need this, read up on DB migration, so you will have to write migration scripts that will help create db instead of syncing